index.html

```diff
@@ -142,6 +142,11 @@
           A successful PATCH request MUST respond with a 2xx status code; the specific code in the 2xx
           range MAY vary according to the response body or request state.
         </p>
+        <p>
+          When receiving a PATCH request that would change the LDP interaction model of a resource
```

Comment:
The essence of this text looks good. I would suggest tying it into the text above (`If an otherwise valid HTTP PATCH request is received that attempts to add statements to a resource that a server disallows...`) by moving the new text to a subsection 1.2.1, and re-phrasing as:
```
The server MUST disallow a PATCH request that would change the LDP interaction model of a resource
to a type that is not a subtype of the current resource type, the request MUST be rejected
with a 409 Conflict response.
```